### ISSUE TITLE
Need option to format query after pasting from another source

### ISSUE THREAD

**Issue by atishamte** [Sequel-Ace/Sequel-Ace#721]
Need option to format query after pasting from another source as per the query editor settings in preference

**Comment by christatedavies** [comment]
This could be in the form of a bundle, and make use of the api at sqlformat.com:

https://sqlformat.org/api/

If I have time, I'll have a look. One of the things I love about other SQL clients is the "beautify sql" commands.

**Comment by Jason-Morcos** [comment]
There already is a built-in bundle to format a query. It just doesn't happen magically on paste, you have to trigger it.

**Comment by jamesstout** [comment]
> as per the query editor settings in preference

Ah, I think you mean that when you paste into the query window, the settings from this screen are not applied:

<img src="https://user-images.githubusercontent.com/1576190/104904128-cbed1200-59bb-11eb-967f-16576a3c72c8.png" width="500">

Is that what you mean?

If so, there's a max character limit of 15000. If you paste more than 15000 characters, then syntax highlighting is not applied.

However, I just discovered a bug where when less than 15000 char are pasted, not all syntax highlighting settings are applied. I've fixed and will PR shortly.
